pipx/commands.py

```diff
@@ -573,30 +536,36 @@ def reinstall_all(
         pipxrc = Pipxrc(venv_dir)
         uninstall(venv_dir, package, local_bin_dir, verbose)
 
-        package_or_url = pipxrc.get_package_or_url(default=package)
+        if pipxrc.main_package.package_or_url is not None:
+            package_or_url = pipxrc.main_package.package_or_url
+        else:
+            package_or_url = package
+
```

Comment:
how about 
```
 package_or_url = pipxrc.main_package.package_or_url or package
```

Reply:
That works and I can do it if you want, but for paranoia's sake I wanted to check explicitly for `None`.

Reply:
What about 
```
package_or_url = pipxrc.main_package.package_or_url if pipxrc.main_package.package_or_url is not None else package
```